### PR TITLE
Added python cdk

### PR DESCRIPTION
**Pull request by Lavanya0513**
Added python cdk

### CODE REVIEW

File: cdk/python/app/lambdapowertuner_stack.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+from aws_cdk import (
+    # Duration,
+    Stack,
+    # aws_sqs as sqs,
```
Comment (alexcasalboni) [inline cdk/python/app/lambdapowertuner_stack.py:4]: do we need these commented imports?

**Review by alexcasalboni — CHANGES_REQUESTED** [review]
Thanks for contributing 🙏  Just a few minor comments to address 🚀

File: cdk/python/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+% .venv\Scripts\activate.bat
+```
+
+Once the virtualenv is activated, you can install the required dependencies.
+
+```
+$ pip install -r requirements.txt
+```
+
+cdk synth
```
[the remark this answers is not in the document]
Comment (Lavanya0513) [reply cdk/python/README.md:45]: This comment line is present a few lines above

[the remark this answers is not in the document]
Comment (alexcasalboni) [reply cdk/python/README.md:45]: You're right, sorry :) didn't have enough coffee yet!

What we are doing here is putting commands in a dotted list: https://github.com/alexcasalboni/aws-lambda-power-tuning/tree/master/cdk/csharp

So I would move the venv details in a new `Virtualenv setup and requirements` section and standardize the `Building, testing, and deploying the app` section with a dotted list, describing individual commands.

**Review by alexcasalboni — APPROVED** [review]
Awesome 🚀  We're ready to merge this 🎉

**Pull request merged**